README.md

```diff
@@ -28,6 +28,61 @@ let alexa = new Alexa.Router()
 Once you intialize the router, you can either configure `actions` or `dispatch` a HTTP request to be
 routed to the actions you have configured.
 
+### `Router`
+
+#### API
+
+`new Alexa.Router(config)`
+
+#### config
+*Required* <br>
+Type: `object`
```

Comment:
since you capitalized the other ones make sure you capitalize this type and update any lowecase types from before